nav2_behavior_tree/CMakeLists.txt

```diff
@@ -9,12 +9,12 @@ find_package(rclcpp_lifecycle REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(builtin_interfaces REQUIRED)
 find_package(geometry_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
 find_package(nav2_msgs REQUIRED)
 find_package(nav_msgs REQUIRED)
 find_package(behaviortree_cpp_v3 REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(tf2_geometry_msgs REQUIRED)
-find_package(std_msgs REQUIRED)
```

Comment:
we still depend on std_msgs in this package (see L41 below), this line should be restored

Reply:
There were two mentions of `std_msgs` on L9 and L17, just changed one to `sensor_msgs`.

Reply:
I missed the second one!